activerecord/lib/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -580,14 +580,12 @@ def extract_limit(sql_type)
           $1.to_i if sql_type =~ /\((.*)\)/
         end
 
-        def translate_exception_class(e, sql)
-          begin
-            message = "#{e.class.name}: #{e.message}: #{sql}"
-          rescue Encoding::CompatibilityError
-            message = "#{e.class.name}: #{e.message.force_encoding sql.encoding}: #{sql}"
-          end
+        def translate_exception_class(e, sql, binds)
+          message = "#{e.class.name}: #{e.message.force_encoding("UTF-8")}"
```

Comment:
We have coverage for forcing message encoding [here](https://github.com/rails/rails/blob/dc28cf8b0420774ae8e0d4410cdef0098ba34337/activerecord/test/cases/adapter_test.rb#L291), but since there's no more SQL interpolation, do we still need to encode this?